build/cluster.js

```diff
@@ -121,7 +152,7 @@ gulp.task('wait-for-heapster', function(doneFn) {
 
     // constantly query the heapster until it is properly running
     heapsterHealthCheck(function(result) {
-      if (result.length > 0) {
+      if (result === runningState) {
```

Comment:
Hmm... The result is always "Running". There's no other type. You should get rid of the result variable here.